posthog/models/team.py

```diff
@@ -181,6 +182,17 @@ def get_effective_membership_level(self, user_id: int) -> Optional["Organization
                 return None
             return requesting_parent_membership.level
 
+    def timezone_for_charts(self) -> str:
+        """
+        Stopgap function for rolling this feature out
+        """
+        distinct_id = self.organization.members.filter(is_active=True).first().distinct_id
+        if posthoganalytics.feature_enabled(
+            "timezone-for-charts", distinct_id, groups={"organization": self.organization_id}
+        ):
+            return self.timezone
```

Comment:
Where is this set?

Reply:
It's the timezone field you can set on the team settings page.
<img width="654" alt="image" src="https://user-images.githubusercontent.com/1727427/163992744-59228148-e0aa-45a3-a343-233f8626f80e.png">